node/src/index.ts

```diff
@@ -302,6 +303,16 @@ export interface Connection {
    * @param name The name of the table to drop.
    */
   dropTable(name: string): Promise<void>
+
+  /**
+   * Instrument the behavior of this Connection with middleware.
+   *
+   * The middlewares callback implementations will be called in the order they are added.
```

Comment:
```suggestion
   * The middleware will be called in the order they are added.
```
Or maybe `middlewares'`?